src/Services/CurrencyConversionClass.php

```diff
@@ -1,26 +1,41 @@
 <?php
 
 namespace FurnitureStoreApi\Services;
+use FurnitureStoreApi\Exceptions\InvalidCurrencyException;
 
 class CurrencyConversionClass
 {
-    public static function currencyConverter($currency, $price)
+    private static string $currency = 'GBP';
+
+    public static function setCurrency(string $currency): void
+    {
+        if (in_array($_GET['currency'], ['GBP', 'USD', 'EUR', 'YEN']))
+        {
+            self::$currency = $currency;
+        }
+        else
+        {
+            throw new InvalidCurrencyException();
+
+        }
+    }
+    public static function currencyConverter(string $price): string
```

Comment:
It seems odd to be that this would take a currency amount as a string and return a string. Return a string, maybe, but the input should be a float, really.

Reply:
This now takes a float and returns a string, because the number formatting function we use returns a string and that is the only way we can get certain numbers to display as 2 decimal points without rounding up to one decimal point.